activesupport/lib/active_support/core_ext/numeric/time.rb

```diff
@@ -5,6 +5,11 @@
 require 'active_support/core_ext/date/acts_like'
 
 class Numeric
+  # Convert number to local Time.
+  def to_time
+    Time.at(self)
```

Comment:
@matthewd we support `to_time` on `String` which can be pretty much anything - in comparison `to_time` on `Numeric` is positively well defined. :smile:

@iamvery `to_time` always returns a local time in the other places it's defined - `in_time_zone` is for `AS::TWZ`.